ansible/roles/test/files/ptftests/copp_tests.py

```diff
@@ -108,6 +108,23 @@ def cancel_timeout(self):
             self.timeout_thr = None
 
     def copp_test(self, packet, count, send_intf, recv_intf):
+        '''
+        Pre-send some packets for a second to absorb the CBS capacity.
+        '''
+        if self.needAbsorbCBS:
+            timeout_start = time.time()
+            sendInFirst = 0
+            while time.time() < timeout_start + 1:
+                testutils.send_packet(self, send_intf, packet)
+                sendInFirst += 1
+            '''
+            count_matched_packets() takes a long time (more than 1 second)
+            Suspend counting matched packetes till all testing packets are sent.
+            Copy the received packets to 'queue'
+            '''
```

Comment:
Use timeout parameter of count matched packets method. It will reduce the waiting time to any time you want
https://github.com/p4lang/ptf/blob/master/src/ptf/testutils.py#L2469

Reply:
I didn't know the timeout can be < 1.  Now I use 0.01 as timeout when calls count_matched_packets()